lib/runtime/src/ops/combine.cc

```diff
@@ -13,322 +13,112 @@
  * limitations under the License.
  */
 
-#include "parallel_ops/combine.h"
+#include "combine.h"
 #include "kernels/combine_kernels.h"
 #include "utils/hash-utils.h"
 
 namespace FlexFlow {
 // declare Legion names
-using Legion::ArgumentMap;
 using Legion::Context;
-using Legion::coord_t;
-using Legion::Domain;
-using Legion::FutureMap;
-using Legion::IndexLauncher;
-using Legion::LogicalPartition;
-using Legion::LogicalRegion;
-using Legion::Machine;
-using Legion::Memory;
 using Legion::PhysicalRegion;
-using Legion::Predicate;
-using Legion::Rect;
-using Legion::RegionRequirement;
 using Legion::Runtime;
 using Legion::Task;
-using Legion::TaskArgument;
-using Legion::TaskLauncher;
 
 using namespace FlexFlow::Kernels::Combine;
 
-CombineParams Combine::get_params() const {
-  CombineParams params;
-  params.combine_legion_dim = this->combine_dim;
-  params.combine_degree = this->combine_degree;
-  return params;
-}
+enum Slots { INPUT, OUTPUT, PROFILING };
```

Comment:
Why not `ATTRS`?

Reply:
I don't use any of the attrs properties, so I don't need to get or bind them (I think)